### DISCUSSION TITLE
How to upgrade to version 2.5.255 on Heroku?

### DISCUSSION THREAD

**Discussion by Vandinho** [requarks/wiki#4767] [Help / Questions]
Can someone direct me an instructional documentation for upgrading the wiki.js implemented in Heroku to the latest version?

In the documentation link there is instruction for Docker, Linux and Windows, but I don't know how to perform the update on Heroku.

https://docs.requarks.io/install/upgrade

**Comment by MaxRickettsUy** [comment]
In case you or anyone else is still interested in this, it worked for me (running `2.5.274` atm).

1. From the heroku cli: `heroku git:clone -a <app-name>`
2. You should get an [empty repository warning](https://help.heroku.com/XOBUHLKQ/why-do-i-see-a-message-you-appear-to-have-cloned-an-empty-repository-when-using-heroku-git-clone). `cd` into cloned directory
3. Add origin remote pointing to `https://github.com/requarks/wiki-heroku/tree/2.x`. 
4. `git pull origin 2.x`
5. `git push heroku`
